lms/djangoapps/grades/tests/test_grades.py

```diff
@@ -208,7 +208,7 @@ def setUp(self):
         course_structure = MagicMock()
         course_structure.get_children = lambda location: locations_to_scored_children[location]
         self.course_grade = CourseGrade(student=None, course=None, course_structure=course_structure)
-        self.course_grade.locations_to_scores = weighted_scores
+        self.course_grade.locations_to_weighted_scores = weighted_scores
```

Comment:
Too fast of a grok! @sanfordstudent it was actually this line causing the test failures. `self.course_grade.locations_to_weighted_scores` was never getting initialized properly.